script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -407,37 +434,44 @@ No, actually, please leave right now――from my house――before Akiha finds
 [sha:d99b71bfa9b00465cf5dbbb717b1953301c937fe]{
 -- Page 384, Offset 15737.
 -- 「あら。あとの半分、真面目な方の話に興味はないの？
-"My, my. No interest in the other, more serious part of why I'm here then?
+"Oh my. So no interest in the other, more serious, half of why I'm here then?
 }
 [sha:a771f758efaed2addb38e8a82821c382787413ee]{
 -- Page 384, Offset 15738.
 -- 敵を倒したら帰るはずのわたしがどうして残っているのか、おかしいと思わない？」
-Do you not find it strange that I'm still here, even though I was supposed to leave after defeating the enemy?"
+// Restructured slightly so that it more clearly reflects off 15735.
+Aren't you a little curious %{i}why%{/i} I'm still here,
+ given that I was supposed to leave after the enemy was defeated?"
 }
 [sha:f0ce5efb0ac6cedb022bb464c4c5e456349e3072]{
 -- Page 385, Offset 15739.
 -- 「……それは……」
-"That's――"
+"That's―――"
 }
 [sha:43e864b897528a57db74fbe9677abf21050e0442]{
 -- Page 385, Offset 15740.
 -- ……そうだった。コイツは気まぐれで街に残るようなヤツじゃない。自由奔放に見えて合理性の塊なんだ。
-True... She's not the kind of person who'd still be around in the city on a whim. She might look like a free-spirit, but she's actually an exemplar of logic and reason.
+// Vampire of FACTS and LOGIC
+I %{i}am%{/i} a little curious...
+ She's not the sort of person to hang around the city on a whim.
+ She may seem like a free spirit, but she's actually an exemplar of logic and reason.
 }
 [sha:8a6e360ff0c0faf3163671721091fb6272932fa9]{
 -- Page 386, Offset 15741.
 -- アルクェイドがいるという事は、まだ倒すべき吸血鬼がいる事を意味している。
-The fact that Arcueid is still here means there are still vampires left to defeat.
+If Arcueid is still here, then that must mean there are still vampires left to defeat.
 }
 [sha:7cb6e81643e246e59f4ecaf74c00290a597eebbd]{
 -- Page 387, Offset 15742.
 -- 「じゃあ、おまえもヴローヴの残党狩りを？」
-"So, you're also after the remnants of Vlov's offspring?"
+"Then, are you also sticking around to clean up what remains of Vlov's offspring?"
 }
 [sha:893f2d7242a684513de5772dd92cf97f1b48e6a6]{
 -- Page 387, Offset 15743.
 -- 「ええ、残った下級死徒の始末よ。ヴローヴとは関係のない話だけどね」
-"Indeed, I'm taking care of the petty Dead Apostle that remain. Not that it has anything to do with Vlov."
+// Assuming this is singular (one Roa)? Is he really a lesser?
```

Comment:
Roa is hard to classify, but yes, he's formally Rank VI because he hasn't been allowed to go any further by the Ancestors. (Ancestor permission is required to ascend beyond Rank VI).

That said, I'm fairly certain this isn't referring to Roa (remember, Arc just killed him 1 day before), but is rather actually a generic statement about the remaining small time Dead Apostles that remain (and ultimately what Shiki and Arc go fight later on). It just so happens Nasu used the same word for them as he does for Rank VIs.

Which is why I translated it the way it was, obviously. Seems I missed a plural, though!

Reply:
Cool, I'll leave a comment to that effect.